src/app.ts

```diff
@@ -30,22 +30,27 @@ app.use(cors(corsOptions));
 
 const initDb = () => {
     return sequelize.sync({ force: true }).then(async () => {
-        const jane = new User({
+        const jane = await User.create({
```

Comment:
nop, on garde le new User :) je préfère largement ce norme à l'autre

Reply:
Pour quel raison ? J'ai changé car ça ne respecte pas la doc :
![image](https://github.com/We-Are-Swissborg/WASB-Back/assets/101713837/899123ae-cb8a-4ac1-be63-972b5913b55c)
https://sequelize.org/docs/v6/core-concepts/model-instances/#incrementing-and-decrementing-integer-values
 
 Sinon y a build je t'avais dis, si ta besoin de faire des modifications avant de save